package.json

```diff
@@ -35,7 +35,7 @@
     "chalk": "^4.1.0",
     "cli-progress": "^3.4.0",
     "commander": "^5.1.0",
-    "stream-chat": "^3.7.0"
+    "stream-chat": "^6.8.0"
```

Comment:
I think this should be removed, now that there is a peer dependency

Reply:
Yes, good point I'll do it and re-test